packages/components/psammead-most-read/src/Rank/index.jsx

```diff
@@ -0,0 +1,173 @@
+  },
+};
+
+const doubleDigitWidth = ({ service }) => {
+  const overrideService = ['bengali', 'arabic'];
+
+  return overrideService.includes(service)
+    ? doubleDigitOverride[service]
+    : doubleDigitDefault;
+};
+
+const StyledWrapper = styled.div`
+  @media (max-width: ${GEL_GROUP_0_SCREEN_WIDTH_MAX}) {
+    min-width: ${props =>
+      listHasDoubleDigits(props) ? doubleDigitWidth(props).group0 : 'auto'};
+  }
+
+  @media (min-width: ${GEL_GROUP_1_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_1_SCREEN_WIDTH_MAX}) {
+    min-width: ${props =>
+      listHasDoubleDigits(props) ? doubleDigitWidth(props).group1 : 'auto'};
+  }
+
+  /* different number order for when css grid is not supported  */
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+    min-width: ${props =>
+      columnIncludesDoubleDigits(props, false)
+        ? doubleDigitWidth(props).group2
+        : 'auto'};
+  }
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MAX}) {
+    min-width: ${props =>
+      columnIncludesDoubleDigits(props, false)
+        ? doubleDigitWidth(props).group3
+        : 'auto'};
+  }
+
+  @supports (${grid}) {
+    @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+      min-width: ${props =>
+        columnIncludesDoubleDigits(props, true)
+          ? doubleDigitWidth(props).group2
+          : 'auto'};
+    }
+
+    @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MAX}) {
+      min-width: ${props =>
+        columnIncludesDoubleDigits(props, true)
+          ? doubleDigitWidth(props).group3
+          : 'auto'};
+    }
+  }
+
+  @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
+    min-width: ${props =>
+      props.listIndex === 5 && listHasDoubleDigits(props)
+        ? doubleDigitWidth(props).group5
+        : 'auto'};
+  }
+`;
+
+const StyledSpan = styled.span`
+  ${({ service }) => getSerifLight(service)}
+  ${({ script }) => script && getFoolscap(script)};
+  color: ${C_POSTBOX};
+  margin: 0; /* Reset */
+  padding: 0;
+  float: ${props => (props.dir === 'rtl' ? 'right' : 'left')};
+`;
+
+const serviceNumerals = service => {
+  const servicesNonWesternNumerals = {
+    arabic: EasternArabic,
+    bengali: Bengali,
+    burmese: Burmese,
+    pashto: EasternArabic,
+    persian: EasternArabic,
+    urdu: EasternArabic,
+  };
+  return servicesNonWesternNumerals[service]
+    ? servicesNonWesternNumerals[service]
+    : WesternArabic;
+};
+
+const MostReadRank = ({ service, script, listIndex, numberOfItems, dir }) => {
+  const numerals = serviceNumerals(service);
+  const rank = numerals[listIndex];
+  return (
+    <StyledWrapper
+      listIndex={listIndex}
+      service={service}
+      numberOfItems={numberOfItems}
+      dir={dir}
+    >
+      <StyledSpan service={service} script={script} dir={dir}>
```

Comment:
Having just the StyledWrapper have `dir` should be sufficient, since StyledSpan inherits this. 
```suggestion
      <StyledSpan service={service} script={script}>
```